### PR TITLE
Fix liveness probe

### PR DESCRIPTION
**Pull request by lightnet328**
Currently, This chart uses command to health checking, but it is not valid.
Therefore, every time a health check is performed, the container restarts.
I think that health checking should use http(s) request.

### CODE REVIEW

File: contrib/helm-charts/portus/templates/portus-deployment.yaml [hunk 1]
```diff
@@ -30,11 +30,14 @@ spec:
...
-                - /api/v1/health
-            initialDelaySeconds: 240
-            periodSeconds: 5
+            httpGet:
+              {{- if .Values.portus.tls.enabled }}
+              scheme: HTTPS
+              {{- end }}
+              path: /api/v1/health
+              port: {{ .Values.portus.service.port }}
+            initialDelaySeconds: 300
```
Comment (stefsuse) [inline contrib/helm-charts/portus/templates/portus-deployment.yaml]: Were you seeing failures with the lower timeout values? 5 minutes and 10 seconds respectively seem too long.

**Review by stefsuse — CHANGES_REQUESTED** [review]
I think you're right. 

Please update Chart.yaml version

File: contrib/helm-charts/portus/templates/portus-deployment.yaml (hunk 1 above)
Comment (lightnet328) [reply contrib/helm-charts/portus/templates/portus-deployment.yaml]: As I measured it just now, Portus took about 7 seconds to return the response in my environment.
```
$ curl https://registry.example.com/api/v1/health -s -o /dev/null -w "Total: %{time_total}\n"
Total: 6.711
```
I will need to validate the proper value of `initialDelaySeconds`. I will work on it later.

**Comment by lightnet328** [comment]
Since `/api/v1/health` returns status code `503` when registry is not registered, liveness probe had failed.
http://port.us.org/features/health.html#health-check
Because I thought that i should use `/api/v1/_ping` to check liveness, I changed it like that.

**Comment by stefsuse** [comment]
Agreed - I did a quick test myself and _ping seems like the appropriate endpoint. 
Please revert timeout values - 240 initial delay is enough.

Chart.yaml has a version key. Update the build version for this change.

**Review by stefsuse — APPROVED** [review]
LGTM and tests passed

**Pull request merged**